src/odin/markup_formatter.cc

```diff
@@ -60,13 +60,13 @@ MarkupFormatter::FormatPhonemeElement(const std::unique_ptr<baldr::StreetName>&
           FormatPhonemeElement(street_name->value(), street_name->pronunciation());
 
       // If the markup string exists then return the street name with the phoneme
-      return boost::make_optional(!phoneme_markup_string.empty(), phoneme_markup_string);
+      return phoneme_markup_string.empty() ? std::nullopt : std::make_optional(phoneme_markup_string);
```

Comment:
is it actually necessary to call `make_optional`? i think we can just return the string and it will be implicitly converted (shorter code for the win)

Reply:
or maybe because its in a ternary it has to have the same type as the other part of the ternary.. mental compiler says probably that